eng/common/dotnet-install.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
```

Comment:
All the three new `dotnet-install` files added are copied over from roslyn repo.